### ISSUE TITLE
SQLAPI: still usable today?

### ISSUE THREAD

**Issue by Vellric** [AnthonyAMC/Public-Denizen-Scripts#77]
Hey Anthony.
I was wondering if your SQLAPI is still usable today, since the last update of it is ~2 years ago. 
In the case that the script is still usable, do you suggest to have some sort of a caching system for preventing lag?

Thanks!
V

**Comment by AnthonyAMC** [comment]
I have no idea if SQLAPI still functions. I haven't touched it in ages. It wasn't even 100% complete when i stopped working on it, though most functionality was there. Give it a try and let me know if it does or doesn't function. Maybe i can fix things that don't work.

**Comment by Vellric** [comment]
After giving it a try, it seems like there is a problem after doing the "/sqlapi --restart" command once I've correctly put the credentials for my database. 

[19:15:08 INFO]: Invalid json: malformed JSON at line 1 column 3 <--- this message is also what the player who use the "/sqlapi --restart" command receive.
[19:15:08 INFO]: [SQPAPI] Restarting System!

That's the only thing I have on the console. Note that the restarting message was send at 19:15, and the time is now 19:26.

I've checked if there was any data on the database I've created for the try of the API, and it is still empty.
